gitignore.js

```diff
@@ -6,13 +6,6 @@ import gitIgnore from 'ignore';
 import slash from 'slash';
 import {toPath} from './utilities.js';
 
-const DEFAULT_IGNORE = [
-	'**/node_modules/**',
-	'**/flow-typed/**',
-	'**/coverage/**',
-	'**/.git',
-];
```

Comment:
While we're removing the ignore option for `isGitIgnored`, I think `globby` should still ignore these paths by default in the main globby methods.

Reply:
These were ONLY used  when glob `.gitignore` file, changing `globby()` to ignore files in these directories is a new breaking change to the `globby()` function, it out scope of this PR.

And if we decide to ignore them by default, but how to unignore them?

Reply:
Ah, you are right, misunderstood.